meteor/client/ui/Shelf/ColoredBoxPanel.tsx

```diff
@@ -0,0 +1,61 @@
+import * as React from 'react'
+import * as _ from 'underscore'
+import {
+	DashboardLayoutColoredBox,
+	RundownLayoutBase,
+	RundownLayoutColoredBox,
+} from '../../../lib/collections/RundownLayouts'
+import { MeteorReactComponent } from '../../lib/MeteorReactComponent'
+import { RundownPlaylist } from '../../../lib/collections/RundownPlaylists'
+import { dashboardElementPosition } from './DashboardPanel'
+import { RundownLayoutsAPI } from '../../../lib/api/rundownLayouts'
+import { Translated } from '../../lib/ReactMeteorData/ReactMeteorData'
+import { withTranslation } from 'react-i18next'
+
+interface IColoredBoxPanelProps {
+	visible?: boolean
+	layout: RundownLayoutBase
+	panel: RundownLayoutColoredBox
+	playlist: RundownPlaylist
+}
+
+interface IState {}
+
+interface IColoredBoxPanelTrackedProps {
+	name?: string
+}
+
+export class ColoredBoxPanelInner extends MeteorReactComponent<
```

Comment:
Why is this exported? This repeats a lot.

Reply:
Addressed by https://github.com/tv2/tv-automation-server-core/commit/72c884713bdf4fb5ee9786f24a85778266c0f2b2